libpod/version.go

```diff
@@ -0,0 +1,50 @@
+package libpod
+
+import (
+	"runtime"
+	"strconv"
+
+	"github.com/projectatomic/libpod/cmd/podman/ioprojectatomicpodman"
+	podmanVersion "github.com/projectatomic/libpod/version"
+)
+
+// Overwritten at build time
+var (
+	// GitCommit is the commit that the binary is being built from.
+	// It will be populated by the Makefile.
+	GitCommit string
+	// BuildInfo is the time at which the binary was built
+	// It will be populated by the Makefile.
+	BuildInfo string
+)
+
+//VersionOutput is an output struct for varlink
+type VersionOutput struct {
+	ioprojectatomicpodman.VarlinkInterface
+	Version   string
+	GoVersion string
+	GitCommit string
+	Built     int64
+	OsArch    string
+}
+
+// CreateVersionOutput returns a VersionOutput struct for varlink and podman
+func CreateVersionOutput() (VersionOutput, error) {
```

Comment:
Rename to Version

Reply:
Actually, rename to GetVersion, and rename the struct it returns to Version

Reply:
done